### PR TITLE
add working group `wg-key-visualizer`

### PR DESCRIPTION
**Pull request by andylokandy**
Signed-off-by: andylokandy <andylokandy@hotmail.com>

## Goals

Implement a visual component to show key heatmap of the TiKV cluster, which is useful for troubleshooting and reasoning inefficient application usage patterns.

### CODE REVIEW

File: working-groups/wg-key-visualizer.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+## Work and Meeting
+
+* Regular Working Group Meeting: 4pm on every Thursday
+* Meeting Zoom: [https://pingcap.zoom.com.cn/j/669899354](https://pingcap.zoom.com.cn/j/669899354)
+* [Work Status & Materials](https://docs.google.com/document/d/1LdvS0oWo79eTINHCZAdI0z0vKGP2xiCLnHYKRHBwfh8)
+
+## Mentors
+
+* [samyubowei](https://github.com/samyubowei), www.58.com
```
Comment (winkyao) [inline working-groups/wg-key-visualizer.md]:
```suggestion
* [samyubowei](https://github.com/samyubowei)([@www.58.com](www.58.com))
```

File: working-groups/wg-key-visualizer.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+* [nolouch](https://github.com/nolouch), [PingCAP](https://github.com/pingcap)
```
Comment (winkyao) [inline working-groups/wg-key-visualizer.md]: ditto

File: working-groups/wg-key-visualizer.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+## Members
+
+* [sivagao](https://github.com/sivagao), [PingCAP](https://github.com/pingcap)
```
Comment (winkyao) [inline working-groups/wg-key-visualizer.md]: ditto

Comment (andylokandy) [reply working-groups/wg-key-visualizer.md]: done

File: working-groups/wg-key-visualizer.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
+# Key Visualizer Working Group
+
+## Goals
+
+Implement a visual component to show key heatmap of the TiKV cluster, which is useful for troubleshooting and reasoning inefficient application usage patterns.
```
Comment (dcalvin) [inline working-groups/wg-key-visualizer.md]:
```suggestion
The Key Visualizer Working Group aims to implement a visual component to show key heatmap of the TiKV cluster, which is useful for troubleshooting inefficient application usage patterns.
```

File: working-groups/wg-key-visualizer.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+## Work and Meeting
+
+* Regular Working Group Meeting: 4pm on every Thursday
+* Meeting Zoom: [https://pingcap.zoom.com.cn/j/669899354](https://pingcap.zoom.com.cn/j/669899354)
+* [Work Status & Materials](https://docs.google.com/document/d/1LdvS0oWo79eTINHCZAdI0z0vKGP2xiCLnHYKRHBwfh8)
+
+## Mentors
+
+* [samyubowei](https://github.com/samyubowei)([@58.com](www.58.com))
+* [nolouch](https://github.com/nolouch), [PingCAP](https://github.com/pingcap)
```
Comment (winkyao) [inline working-groups/wg-key-visualizer.md]:
```suggestion
* [nolouch](https://github.com/nolouch), [@PingCAP](https://github.com/pingcap)
```

File: working-groups/wg-key-visualizer.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+## Members
+
+* [sivagao](https://github.com/sivagao), [PingCAP](https://github.com/pingcap)
```
Comment (winkyao) [inline working-groups/wg-key-visualizer.md]:
```suggestion
* [sivagao](https://github.com/sivagao), [@PingCAP](https://github.com/pingcap)
```

File: working-groups/wg-key-visualizer.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+* [andylokandy](https://github.com/andylokandy), [PingCAP](https://github.com/pingcap)
```
Comment (winkyao) [inline working-groups/wg-key-visualizer.md]:
```suggestion
* [andylokandy](https://github.com/andylokandy), [@PingCAP](https://github.com/pingcap)
```

File: working-groups/wg-key-visualizer.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+* [hundundm](https://github.com/hundundm), [PingCAP](https://github.com/pingcap)
```
Comment (winkyao) [inline working-groups/wg-key-visualizer.md]:
```suggestion
* [hundundm](https://github.com/hundundm), [@PingCAP](https://github.com/pingcap)
```

File: working-groups/wg-key-visualizer.md [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+* [zzh-wisdom](https://github.com/zzh-wisdom), [HUST](http://www.hust.edu.cn/)
```
Comment (winkyao) [inline working-groups/wg-key-visualizer.md]:
```suggestion
* [zzh-wisdom](https://github.com/zzh-wisdom), [@HUST](http://www.hust.edu.cn/)
```

File: working-groups/wg-key-visualizer.md [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+* [Sullivan12138](https://github.com/Sullivan12138), [HUST](http://www.hust.edu.cn/)
```
Comment (winkyao) [inline working-groups/wg-key-visualizer.md]:
```suggestion
* [Sullivan12138](https://github.com/Sullivan12138), [@HUST](http://www.hust.edu.cn/)
```

**Pull request merged**